lib/videos_list/view/videos_list_page.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+import 'package:my_videos_player/app/view/app.dart';
+import 'package:my_videos_player/l10n/l10n.dart';
+
+class VideosListPage extends StatelessWidget {
+  const VideosListPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return const VideosListView();
+  }
+}
+
+class VideosListView extends StatelessWidget {
```

Comment:
Is this used anywhere else except tests? If not it should be marked with "@visibleForTesting" attribute.